trace.go

```diff
@@ -409,6 +409,12 @@ func (e *TraceErr) GetFields() map[string]interface{} {
 	return e.Fields
 }
 
+// Unwrap returns wrapped error. The returned error may also wrap another one,
```

Comment:
Nit: This reads a bit weird, giving the opposite impression - that it wraps returned error. Maybe

`Unwrap returns the error this trace error wraps. The returned error ...`?